hkmc2/shared/src/test/mlscript/decls/Prelude.mls

```diff
@@ -42,10 +42,12 @@ declare module Predef with
     val x = 0
   class MatchResult(captures)
   class MatchFailure(errors)
-  class __EffectSig(next, tail, handleBlockList, resumed, handler, handlerFun)
-  class __Return(value)
+
+declare module Runtime with
+  class EffectSig(contTrace, handler, handlerFun)
+  class Return(value)
   // This declaration is not used currently except for experimentation
-  abstract class __StackDelay with
+  abstract class StackDelay with
     fun perform()
```

Comment:
Let's not do that. `Prelude.mls` is for listing built-in things that either come with JavaScript or that represent MLscript base concepts without a JS counterpart (like `Int`).

We should also remove `Predef` from here, for what it's worth. It has nothing to do here.